Podfile

```diff
@@ -219,28 +219,28 @@ abstract_target 'Apps' do
 
   pod 'Automattic-Tracks-iOS', '~> 0.13'
   # While in PR
-  # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :branch => ''
+  # pod 'Automattic-Tracks-iOS', git: 'https://github.com/Automattic/Automattic-Tracks-iOS.git', branch: ''
   # Local Development
-  # pod 'Automattic-Tracks-iOS', :path => '~/Projects/Automattic-Tracks-iOS'
+  # pod 'Automattic-Tracks-iOS', path: '~/Projects/Automattic-Tracks-iOS'
 
   pod 'NSURL+IDN', '~> 0.4'
 
   pod 'WPMediaPicker', '~> 1.8.7'
-  # pod 'WPMediaPicker', :git => 'https://github.com/wordpress-mobile/MediaPicker-iOS.git', :tag => '1.7.0'
+  # pod 'WPMediaPicker', git: 'https://github.com/wordpress-mobile/MediaPicker-iOS.git', tag: '1.7.0'
```

Comment:
Actually, we should delete this because it adds little to what we can already do with the standard syntax.

I suppose it might have been useful in the past, before we had our own specs `source` repository.

Reply:
```suggestion
```